earthpy/spatial.py

```diff
@@ -370,6 +370,107 @@ def crop_image(raster, geoms, all_touched=True):
     return out_image, out_meta
 
 
+def crop_all(
+    band_paths,
+    output_dir,
+    geoms,
+    overwrite=False,
+    all_touched=True,
+    verbose=True,
+):
+    """Takes a list of rasters and a boundary, and crops them efficiently.
+
+    Parameters
+    ----------
+    band_paths : list of file paths
+        List of paths of bands that are to be cropped.
+    output_dir : list or string
+        If a list is provided, then the output files from this function will be written to
```

Comment:
How about something like:
```
Provide a single directory path if you wish to specify the location of the output cropped files only. _crop will be appended to the file name for each output cropped image.
To customize output file names and locations, provide a list of full paths with file names. The number of paths in the list must match the number of files being cropped. 
```

Reply:
please note much of this section is too many characters wide. please fix the line width by adding returns where needed!

Reply:
My PyCharm environment shows the return line way to the right, so I can't really tell where the returns are supposed to be, sorry! I took a guess and added a bunch where I think the line likely is.

Reply:
@nkorinek  you can setup the editor to have a line at the PEP 8 recommended length. that is what i have setup! remind me to show you on thursday.